torch/csrc/jit/passes/memory_planning/memory_observer.cpp

```diff
@@ -0,0 +1,165 @@
+//
+// Created by Maksim Levental on 9/11/21.
```

Comment:
lmao with the 9/11 citation... haven't yet reviewed this class, waiting on description of what it does/purpose and anything in particular you would like me to think about when reviewing

Reply:
whoops!